### PR TITLE
Added gulp-watch

### PR DESCRIPTION
**Pull request by lipis**
Related #580

### CODE REVIEW

**Comment by lipis** [comment]
Can you do.. `gulp reset` and then `npm install` to have a clean install before running `gulp` again..?

File: gulp/tasks/watch.coffee [hunk 1]
```diff
@@ -5,21 +5,29 @@ paths = require '../paths'
...
 
 gulp.task 'watch', false, ->
-  gulp.watch 'requirements.txt', ['pip']
-  gulp.watch 'package.json', ['npm']
-  gulp.watch 'bower.json', ['ext_watch_rebuild']
-  gulp.watch 'gulp/config.coffee', ['ext:dev', 'style:dev', 'script:dev']
-  gulp.watch paths.static.ext, ['ext:dev']
-  gulp.watch "#{paths.src.script}/**/*.{coffee,js}", ['script:dev']
-  gulp.watch "#{paths.src.style}/**/*.less", ['style:dev']
+  $.watch 'requirements.txt', ->
```
Comment (topless) [inline gulp/tasks/watch.coffee]: These can be in one line.

**Comment by ventice** [comment]
@lipis, I did `gulp reset` and `npm install` and got exactly the same result in terms of warnings and multiple processing of ext. Anyhow, despite the warnings looks like the watching works and node.exe stays at 0% of CPU. In any case, I would stay with my polling solution rather than with this one but with warnings (:

**Comment by topless** [comment]
@ventice what are you `node` and `npm` versions?

**Comment by ventice** [comment]
>npm --version
2.15.9

>node --version
v4.6.1

**Comment by topless** [comment]
[15:45:24] $ npm -v
3.10.3

[15:51:16] $ node -v
v6.6.0

**Comment by ventice** [comment]
strange, I upgraded both just three days ago.

**Comment by topless** [comment]
Do you use nave or any other virtual environment for sandboxing dependencies?

**Comment by ventice** [comment]
Actually, this is the last installer available on the node site:
https://nodejs.org/dist/v4.6.1/node-v4.6.1-x64.msi

**Comment by ventice** [comment]
I don't use nave or any other sandbox

**Comment by ventice** [comment]
UPD: After update to node v6.9.1 and npm 3.10.8 the warning are gone. node.exe stays at 0%. Fine job!

**Comment by ventice** [comment]
On subsequent launches warnings reappeared in some shortened form but watching keeps working.

**Pull request merged**